.github/workflows/linux_builds.yml

```diff
@@ -36,9 +36,7 @@ jobs:
             tests: true
             sconsflags: float=64 use_asan=yes use_ubsan=yes
             proj-test: true
-            # Can be turned off for PRs that intentionally break compat with godot-cpp,
-            # until both the upstream PR and the matching godot-cpp changes are merged.
```

Comment:
Ah too late but I would have kept the comment as it's bound to be needed again the future. We can readd it then.